src/ecdsa.js

```diff
@@ -9,9 +9,36 @@ var ZERO = new Buffer([0])
 var ONE = new Buffer([1])
 
 // https://tools.ietf.org/html/rfc6979#section-3.2
-function deterministicGenerateK(curve, hash, d) {
+function deterministicGenerateK(curve, hash, d, checkSig) {
   typeForce('Buffer', hash)
   typeForce('BigInteger', d)
+//  typeForce('Function', checkSig)
```

Comment:
Move this to the comment 2 lines done so peeps won't wonder why this is commented out